sdk/react/panels/videoView.js

```diff
@@ -121,10 +121,13 @@ class VideoView extends React.Component {
     }
   };
 
-  onSeekPressed = (isForward) => {
-    let configSeekValue = (isForward) ? this.props.config.skipControls.skipForwardTime : this.props.config.skipControls.skipBackwardTime;
-    configSeekValue = Utils.restrictSeekValueIfNeeded(configSeekValue);
-    const seekValue = (isForward) ? configSeekValue : configSeekValue * (-1);
+  onSeekPressed = (skipCountValue) => {
+    if (skipCountValue == 0) { return null; }
+
+    let configSeekValue = (skipCountValue > 0) ? this.props.config.skipControls.skipForwardTime : this.props.config.skipControls.skipBackwardTime;
```

Comment:
Can we use ```const``` instead ```let```?

Reply:
I didn't change this part, so it was already `let`.

Reply:
If we keep the `restrictSeekValueIfNeeded` disabled then we can use `const`.